Lesson23/app/baljan/styles.css

```diff
@@ -0,0 +1,125 @@
+:root {
+  --primary-color: #4caf50;
+  --secondary-color: #333;
+  --font-family: Arial, sans-serif;
+  --font-size-large: 18px;
+  --font-size-medium: 16px;
+  --font-size-small: 14px;
+  --border-radius: 4px;
+  --transition-duration: 0.3s;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: var(--font-family);
+  text-align: center;
+}
+
+h1 {
+  color: var(--secondary-color);
+}
+
+.controls {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  margin-bottom: 20px;
+}
+
+label {
+  padding: 10px 20px;
+  font-size: var(--font-size-large);
+  font-weight: bold;
+}
+
+.select,
+.button {
+  width: 200px;
+  height: 40px;
```

Comment:
```suggestion
```

Avoid adding height for elements like this containing text. It's better to make them as high as they need to be (line hight + padding  + border)